src/kdc/kdc_preauth.c

```diff
@@ -1349,29 +1340,37 @@ return_padata(krb5_context context, krb5_kdcpreauth_rock rock,
                 }
             }
         }
+        send_pa = NULL;
```

Comment:
This can happen in the declaration block; it's not used before here.

Reply:
But we're inside a loop, and we need to make sure it's null before the call on each iteration.

Reply:
Whoops, yes, you're right.